Jenkinsfile-Release

```diff
@@ -63,6 +65,11 @@ dockerizedBuildPipeline(
       runSafely 'git add charts'
     }
 
+    if ('nexus-iq-server-ha'.equals(params.chart)) {
+      runSafely "curl -L 'https://github.com/sonatype/nexus-iq-server-ha/blob/main/docs/${nexusIqServerHaArtifactName}?raw=true' -o 'docs/${nexusIqServerHaArtifactName}'"
+      runSafely './build-nexus.sh'
```

Comment:
Was curious if running `./build-nexus.sh` is correct but it's just running
```
cd docs
helm repo index . --url https://sonatype.github.io/helm3-charts
```
which should be fine for the IQ HA chart too

Reply:
it may be time to rename that `build-nexus.sh` to `build-index.sh` or something, since it's taken on a generic use.

Reply:
alternatively, you could `curl` the HA chart first, then `upgrade.sh` and `build-iq.sh` and let that build the index once.

Reply:
> alternatively, you could `curl` the HA chart first, then `upgrade.sh` and `build-iq.sh` and let that script build the index once.

:+1: I adopted this option 960168b